src/Components/OperationsGroup.jsx

```diff
@@ -22,45 +22,48 @@ import useStore from '../store/useStore'
  */
 export default function OperationsGroup({viewer, unSelectItem}) {
   const classes = useStyles()
+  const toggleIsCommentsOn = useStore((state) => state.toggleIsCommentsOn)
   const toggleIsPropertiesOn = useStore((state) => state.toggleIsPropertiesOn)
   const openDrawer = useStore((state) => state.openDrawer)
-  const closeDrawer = useStore((state) => state.closeDrawer)
-  const isDrawerOpen = useStore((state) => state.isDrawerOpen)
   const selectedElement = useStore((state) => state.selectedElement)
 
-  const toggle = () => {
-    isDrawerOpen ? closeDrawer() : openDrawer(),
-    toggleIsPropertiesOn()
+  const toggle = (panel) => {
+    openDrawer()
+    panel === 'Properties' ? toggleIsPropertiesOn() : null
+    panel === 'Notes' ? toggleIsCommentsOn() : null
   }
 
   return (
     <div className={classes.container}>
-      <div className={classes.shareAndIssues}>
+      <div className={classes.topGroup}>
         <ShareControl viewer={viewer}/>
-        <IssuesControl viewer={viewer}/>
-
-
+        <TooltipIconButton
+          title = 'Notes'
+          icon ={<MarkupIcon/>}
+          onClick = {() => toggle('Notes')}
+        />
       </div>
       <div className={classes.lowerGroup}>
         {
           selectedElement ?
           <TooltipIconButton
-            title="Properties"
-            onClick={toggle}
-            icon={<ListIcon/>}/> : null
+            title = "Properties"
+            onClick = {() => toggle('Properties')}
+            icon={<ListIcon/>}/> :
+          null
         }
         {useIsMobile() ?
-         <TooltipIconButton
-           title="Section plane"
-           onClick={() => viewer.clipper.createPlane()}
-           icon={<CutPlaneIcon/>}/> :
+          <TooltipIconButton
+            title = "Section plane"
+            onClick = {() => viewer.clipper.createPlane()}
+            icon = {<CutPlaneIcon/>}/> :
           null
         }
-        <TooltipIconButton title="Clear selection" onClick={unSelectItem} icon={<ClearIcon/>}/>
+        <TooltipIconButton title="Clear selection" onClick = {unSelectItem} icon = {<ClearIcon/>}/>
         <ShortcutsControl/>
       </div>
       {/* Invisible */}
-      <CameraControl viewer={viewer}/>
```

Comment:
good to see consistent style, but i think to minimize change for now we should use no spaces, as in discussion on discorc